src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -219,7 +219,9 @@ public void keyPressed(java.awt.event.KeyEvent e) {
     public synchronized void listen(FieldAddedOrRemovedEvent event) {
         // other field deleted -> update other fields tab
         if (OtherFieldsTab.isOtherField(entryType, event.getFieldName())) {
-            rebuildOtherFieldsTab();
+            DefaultTaskExecutor.runInJavaFXThread(() -> {
```

Comment:
You can actually omit the curly braces and write it as one liner:
`DefaultTaskExecutor.runInJavaFXThread(() ->  rebuildOtherFieldsTab());`